conf/test.config

```diff
@@ -27,4 +27,6 @@ params {
   gtf = 'https://github.com/nf-core/test-datasets/raw/rnaseq/reference/genes.gtf'
   gff = 'https://github.com/nf-core/test-datasets/raw/rnaseq/reference/genes.gff'
   transcript_fasta = 'https://github.com/nf-core/test-datasets/raw/rnaseq/reference/transcriptome.fasta'
+  // test rRNA removal
+  remove_rRNA = true
```

Comment:
This should go to a dedicated extra `test_rrna` profile for example OR be shifted to a separate line in `travis.yml` that tests explicitly whether it works. 

Adding it here will change our current default tests significantly to before, which is not desired I think...

Reply:
Done in commit fb1cd17110895dd24c464762369c2b9fb8803171